### PR TITLE
relay/circuit-v2: Do minor fixes

### PR DESCRIPTION
**Pull request by thomaseizinger**
A few things I noticed while reading through the spec.

Not sure if we want to bump the revision for such minor fixes? Can always drop the last commit!

### CODE REVIEW

File: relay/circuit-v2.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This is the version 2 of the libp2p Circuit Relay protocol.
 
 | Lifecycle Stage | Maturity       | Status | Latest Revision |
 |-----------------|----------------|--------|-----------------|
-| 1A              | DRAFT          | Active | r0, 2021-05-05  |
+| 1A              | DRAFT          | Active | r1, 2021-12-09  |
```
Comment (mxinden) [inline relay/circuit-v2.md:7]: I don't think a version bump is necessary, given that this does not change any functionality, i.e. given that implementations do not have to change based on this.

**Review by mxinden — COMMENTED** [review]
Thanks. One comment, otherwise looks good to me.

File: relay/circuit-v2.md [hunk 2]
```diff
@@ -226,7 +226,7 @@ Reservation {
  `p2p-circuit/p2p/QmPeer` where `QmPeer` is its peer ID.
 - the `voucher` is the binary representation of the reservation voucher -- see [Reservation Vouchers](#reservation-vouchers) for details.
 
-The `limit` field, if present, provides information about the limits applied by the relay in relayed connection. When omitted, it indicates that the relay does not apply any limits.
+The `limit` field in `HopMessage`, if present, provides information about the limits applied by the relay in relayed connection. When omitted, it indicates that the relay does not apply any limits.
 
 The struct has the following fields:
 ```
```
Comment (yusefnapora) [inline relay/circuit-v2.md:232]: One more missing `}` on the def for `Limit` here :)

**Review by yusefnapora — APPROVED** [review]
Hey all, sorry I slept on reviewing the initial draft of this spec. My belated take is that it's really well done and explains the protocol & its motivations clearly. The changes in this PR seem great and non-controversial 😄

File: relay/circuit-v2.md (hunk 1 above)
Comment (thomaseizinger) [reply relay/circuit-v2.md]: Dropped the commit.

**Review by mxinden — APPROVED** [review]
Thanks Thomas.

**Pull request merged**